### ISSUE TITLE
Inventory cannot be added!

### ISSUE THREAD

**Issue by chinwailun** [smashita/CPT341SDA#1]
When sugar and chocolate are added into the inventory, no changes are made to the units. The sugar and chocolate units remain the same.

**Comment by HatsyRei** [comment]
To be precise, the consistent error message is
> Inventory was not added

The few observations I made were as follows:
1) Changes were actually made to the inventory, but only to "coffee" and "milk".
2) If you attempt to input values which are not positive integers to "coffee" or "milk", all changes will not be executed.
3) If you attempt to input values which are not positive integers to "sugar" or "chocolate", changes to "coffee" and "milk" will still register.

Basically, "sugar" and "chocolate" is voided. Input verification doesn't work on the two fields, no changes will register.
